docs/source/developers/cpp/building.rst

```diff
@@ -121,9 +121,21 @@ argument is omitted then a release build will be produced.
 
 .. note::
 
-   You need to more options to build on Windows. See
+   You need to set more options to build on Windows. See
    :ref:`developers-cpp-windows` for details.
 
+Other options for build type are also possible:
+
+* ``Debug``: doesn't apply any compiler optimizations and leaves debugging
+  information in the binary.
+* ``RelWithDebInfo``: applies compiler optimizations while leaving debug
+  information in the binary.
```

Comment:
I suggest to change `leaves debugging information` to `adds debugging information`.